src/rtc/rtc_s1_single_job.py

```diff
@@ -2010,15 +2142,15 @@ def get_radar_grid(geogrid, dem_interp_method_enum, product_id,
     layers_nbands = 1
     shape = [layers_nbands, geogrid.length, geogrid.width]
 
-    incidence_angle_raster = _create_raster_obj(
-        output_dir, product_id, LAYER_NAME_INCIDENCE_ANGLE,
-        gdal.GDT_Float32, shape, radar_grid_file_dict,
-        output_obj_list, save_incidence_angle, extension)
     local_incidence_angle_raster = _create_raster_obj(
         output_dir, product_id, LAYER_NAME_LOCAL_INCIDENCE_ANGLE,
         gdal.GDT_Float32, shape,
         radar_grid_file_dict, output_obj_list, save_local_inc_angle,
         extension)
+    incidence_angle_raster = _create_raster_obj(
+        output_dir, product_id, LAYER_NAME_INCIDENCE_ANGLE,
+        gdal.GDT_Float32, shape, radar_grid_file_dict,
+        output_obj_list, save_incidence_angle, extension)
```

Comment:
I think it would be more readable if the order of `_create_raster_obj()` is consistent with the order of the parameters `save_*` flags in this function. I see your intention as to place the local incidence angle as the first key-value pair in the dictionary `radar_grid_file_dict`. That being case, I think you can swap the locations of `save_incidence_angle` and `save_local_inc_angle` in the function header.